tests/src/GC/Coverage/smalloom.csproj

```diff
@@ -9,6 +9,7 @@
     <OutputType>Exe</OutputType>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
+    <GCStressIncompatible>true</GCStressIncompatible>
```

Comment:
If this runs on a small device with other tests running on other CPU's, you may also want to mark this 
`<IsLongRunningGCTest>true</IsLongRunningGCTest>`  so it only runs serialized and doesn't cause spurious oom failures in other tests.

Reply:
> If this runs on a small device with other tests running on other CPU's, you may also want to mark this <IsLongRunningGCTest>true</IsLongRunningGCTest> so it only runs serialized and doesn't cause spurious oom failures in other tests.

It looks to me like setting `<IsLongRunningGCTest>true</IsLongRunningGCTest>` causes code like this to be added to the test wrapper:
```
if not defined RunningLongGCTests (
  echo Skipping execution because long-running GC tests are not enabled.
  Exit /b 0
)
```
and `RunningLongGCTests` is set by the root runtest.cmd only if "longgc" is passed as an argument.

Looks like netci.groovy does create `longgc` test jobs, but only for x64. It does set RunningLongGCTests, and also sets for sequential execution. But it also runs tests from a playlist (`tests\longRunningGcTests.txt`) (for non-Windows) which would need to be kept up-to-date with the annotations.